frontend/src/scenes/web-analytics/webAnalyticsLogic.ts

```diff
@@ -5,6 +5,7 @@ import api from 'lib/api'
 import { RETENTION_FIRST_TIME, STALE_EVENT_SECONDS } from 'lib/constants'
 import { dayjs } from 'lib/dayjs'
 import { isNotNil } from 'lib/utils'
+import * as tldts from 'tldts'
```

Comment:
This potentially adds about [100KB of JS](https://bundlephobia.com/package/tldts@6.0.22). Might be worth double checking if so... and perhaps just importing `{ getSubdomain }` solves this?

Reply:
I suspect that this doesn't help much, as it'll still need the big list of 3LDs etc. Will try it anyway!